platform-sdk/swirlds-logging/src/main/java/com/swirlds/logging/api/internal/LoggingSystem.java

```diff
@@ -202,4 +281,14 @@ public void stopAndFinalize() {
     public LogEventFactory getLogEventFactory() {
         return logEventFactory;
     }
+
+    /**
+     * Returns the handler list for testing purposes
+     *
+     * @return handler list
+     */
+    @VisibleForTesting
+    List<LogHandler> getHandlers() {
+        return handlers;
```

Comment:
```suggestion
        return Collections.unmodifyableList(handlers);
```
+ remove @VisibleForTesting